### PR TITLE
Feat/hero content image and blog hero image

### PR DESCRIPTION
**Pull request by ikrestov**
cc @moaiandin

### CODE REVIEW

**Comment by tbarn** [comment]
Should we include a slight darkening of the image so we aren't limited by what images we use and so the subtitle is still easily readable? 

Screen Shot 2019-04-08 at 11 48 11 AM

**Comment by moaiandin** [comment]
Hi @tbarn, we are already darkening them, could make it more dark but then it will look a little bit too dark.

**Comment by lottamus** [comment]
image
Looks like it failed to build. Can you try running a `yarn build` locally?

File: src/components/Hero/index.tsx [hunk 1]
```diff
@@ -364,6 +249,145 @@ export const Hero: React.FunctionComponent<IHero> = ({
...
+          
+          {contentBgImage && (
+            <div
+              className={cn('absolute pin z-0')}
+              style={{
+                background: `url("${contentBgImage}") no-repeat 50% 50%`,
+                backgroundSize: 'cover',
+              }}
+            >
+              {contentBgOverlay && (<div
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: I don't think we need this extra div

File: src/components/Hero/index.tsx [hunk 2]
```diff
@@ -364,6 +249,145 @@ export const Hero: React.FunctionComponent<IHero> = ({
             bottom: image ? -200 : 0,
           }}
         />
+
+        <div className={cn(headerHeightClass, 'w-100')} />
+
+        <div className={cn('relative', 'w-100')}>
+          
+          {contentBgImage && (
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: Let's move the contentBgImage outside of the div on L255: https://github.com/stoplightio/stoplight.io/pull/737/files#diff-46589db87698917798337df55156d320R255 so the image fills the header as well

File: src/components/Hero/index.tsx [hunk 3]
```diff
@@ -364,6 +249,145 @@ export const Hero: React.FunctionComponent<IHero> = ({
...
+            <div
+              className={cn('absolute pin z-0')}
+              style={{
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: Change this to be something like:

```
backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("${contentBgImage}")`,
backgroundSize: 'cover',
backgroundRepeat: 'no-repeat',
backgroundPosition: '50% 50%',
```

File: src/components/Hero/index.tsx [hunk 4]
```diff
@@ -364,6 +249,145 @@ export const Hero: React.FunctionComponent<IHero> = ({
             bottom: image ? -200 : 0,
           }}
         />
+
+        <div className={cn(headerHeightClass, 'w-100')} />
+
+        <div className={cn('relative', 'w-100')}>
```
Comment (lottamus) [inline src/components/Hero/index.tsx]: Lets add a text shadow something like:

```
textShadow: rgba(0, 0, 0, 0.5) 1px 1px 0px
```

**Comment by ikrestov** [comment]
Hey,

I have fixed typo breaking build, rebased and addressed comments.

image

Thanks,
Igor.

**Comment by moaiandin** [comment]
@lottamus all fixed and ready to go :)

**Pull request merged**